docs/source/glossaries.rst

```diff
@@ -0,0 +1,107 @@
+**********
+Glossaries
+**********
+
+=========== ==============================================================
+Keyword     Description                                                   
+=========== ==============================================================
+ACK         Acknowledge                                                   
+am          Active Message                                                
+AMO         Atomic Memory Operation                                       
+ANL         Argonne National Laboratory                                   
+AZP         AZure Pipeline                                                
+bcopy       Byte copy                                                     
+Bistro      Binary Instrumentation                                        
+BTL         Byte Transfer Layer                                           
+cm          Connection Manager                                            
+CMA         Cross Memory Attach                                           
+CQ          Completion Queue(Infiniband)                                  
+CQE         Completion Queue Entry(Infiniband)                            
+csmock      static analysis tools                                         
+CUDA        Compute Unified Device Architecture(NVIDIA)                   
+DC          Dynamically Connected transport(Infiniband)                   
```

Comment:
Dynamically Connected transport(Mellanox/Nvidia Infiniband)  
(DC is not part of IBTA spec)

Reply:
done. modified Infiniband -> InfiniBand.